user-org-util/src/main/resources/externalResource.properties

```diff
@@ -0,0 +1 @@
+userOrgServiceLogLevel=INFO
```

Comment:
userOrgServiceLogLevel this env need to be rename as "sunbird_user_org_log_level"